.github/workflows/build.yml

```diff
@@ -11,8 +11,9 @@ jobs:
         fetch-depth: 0
     - name: Install toolchain
       run: |
-        sudo apt install gcc-aarch64-linux-gnu iasl device-tree-compiler python3-pyelftools
+        sudo apt update
```

Comment:
The changes to other files looks fine, but what is this for?

Reply:
Without sudo apt update the build process did not succeed, you can see it in the history.

[https://github.com/jaredmcneill/quartz64_uefi/actions/runs/5146431642/jobs/9265465330](url)

I just messed up to fix it